src/main/groovy/edu/oregonstate/mist/locations/resources/LocationResource.groovy

```diff
@@ -82,9 +82,15 @@ class LocationResource extends Resource {
      * @return
      */
     private List getArcGisAndMapData() {
-        HashMap<String, ArcGisLocation> arcGisLocations = arcGisDAO.getArcGisLocations()
+        // Get arcgis centroid data from http request
```

Comment:
How about having a method in the DAO that merges both arcGisDAO.getArcGisLocations() and locationDAO.addArcGisGeometries() ? Then the concept that there are two different feeds from arcgis are being merged is abstracted away from this method. This method should just have the concept of arcgis and campusmap.